plugins/jobs/src/js/pages/JobRunHistoryTable.js

```diff
@@ -6,6 +6,7 @@ import PropTypes from "prop-types";
 import React from "react";
 import { Icon, Tooltip } from "@dcos/ui-kit";
 import { SystemIcons } from "@dcos/ui-kit/dist/packages/icons/dist/system-icons-enum";
+import { Dropdown } from "reactjs-components";
```

Comment:
why did we chose not to use `ui-kit`s Dropdown?

Reply:
We don't use it anywhere else, I used the service table actions as an example.

Reply:
let's please make use of `ui-kit` as much as we can!

Reply:
If we want to use the ui-kit dropdown, we should use it everywhere for consistency, since it looks and behaves different.